src/app/components/Common/PlaylistPanel/PlaylistPanel.jsx

```diff
@@ -98,23 +99,23 @@ export default class PlaylistPanel extends React.Component {
             <div className={classes.titleWrapper}>
               <span className={classes.name}>{playlist.attributes.name}</span>
               <span className={classes.curator}>
-                {`Playlist by ${playlist.attributes.curatorName}`}
+                {`${translate.playlistBy}${playlist.attributes.curatorName}`}
```

Comment:
The curator name should be passed in as a placeholder. Allows support for languages with different word order (ie where the curator name would appear before the equivalent to 'Playlist by')